includes/embeds/class-amp-twitter-embed.php

```diff
@@ -229,15 +229,31 @@ private function create_amp_twitter_and_replace_node( $dom, $node ) {
 			return;
 		}
 
+		$additional_attributes = [];
+
+		if ( $node->hasAttributes() ) {
+			foreach ( $node->attributes as $attr ) {
+				$name  = $attr->nodeName;
+				$value = $attr->nodeValue;
+
+				if ( substr( $name, 0, 5 ) === 'data-' ) {
```

Comment:
Small improvement: no need for the `$name` or `$value` attributes. Just re-use `$attr->nodeName` here and `$attr->nodeValue` below.

Reply:
Done